internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -164,15 +164,24 @@ func generatePKI(config runtime.Configurator) (err error) {
 	}
 
 	ips = append(ips, stdlibnet.ParseIP("127.0.0.1"))
+	if net.IsIPv6(ips...) {
+		ips = append(ips, stdlibnet.ParseIP("::1"))
+	}
 
 	hostname, err := os.Hostname()
 	if err != nil {
 		return fmt.Errorf("failed to get hostname: %w", err)
 	}
 
+	dnsNames, err := net.DNSNames()
+	if err != nil {
+		return fmt.Errorf("failed to get host DNS names: %w", err)
+	}
+	dnsNames = append(dnsNames, "localhost")
```

Comment:
Looks like linter is mad about this and want a space above this line.

Reply:
Yeah, just fixed.